s3_to_redshift/config.d/chatbot_reports.json

```diff
@@ -7,13 +7,13 @@
   "dbtable": "microservice.chatbot_reports",
   "column_count": 13,
   "columns": [
+    "dashboard_id",
     "conf_score",
     "sentiment_score",
     "sentiment_magnitude",
-    "sessionorigin",
-    "conversation_index",
     "conversation_length",
     "conversation_duration",
+    "sessionorigin",
```

Comment:
Not a big deal, but this is the only column in the table that doesnt have an underscore separating words. If its easy, can this be fixed?